recipes/qt-advanced-docking-system/all/conanfile.py

```diff
@@ -26,70 +35,108 @@ class QtADS(ConanFile):
         "shared": False,
         "fPIC": True,
     }
-    generators = "cmake", "cmake_find_package", "cmake_find_package_multi"
 
-    _cmake = None
-    _qt_version = "5.15.6"
+    @property
+    def _qt_major(self):
+        return Version(self.dependencies["qt"].ref.version).major
 
     @property
-    def _source_subfolder(self):
-        return "source_subfolder"
+    def _min_cppstd(self):
```

Comment:
Is there a reason to force cpp standard here? When building ads dynamically (most likely due to lgpl), it doesn't matter. If build statically, requirement will be forced by Qt anyway.